data/src/main/java/ch/srg/dataProvider/integrationlayer/data/ImageUrl.kt

```diff
@@ -16,6 +16,7 @@ import java.io.Serializable
  */
 @Suppress("SerialVersionUIDInSerializableClass")
 @kotlinx.serialization.Serializable(with = ImageUrlSerializer::class)
+// TODO Why is this class 'java.io.Serializable'?
```

Comment:
Since this PR is already a breaking change, I'll probably remove the fact that this class is a `java.io.Serializable`.
Users of this library can put the url in the `Bundle` directly, and then recreate an `ImageUrl` when reading the `Bundle`.
What do you think?

Reply:
I've removed the `java.io.Serializable` in https://github.com/SRGSSR/srgdataprovider-android/pull/50/commits/7248a47ca4eada3c2e4a9bcb52cf0935c3cc4cf1
https://github.com/SRGSSR/playsrg-android has a custom Room converter for `ImageUrl` anyway, and I didn't see any other use cases where `java.io.Serializable` is necessary.